robolectric/src/main/java/org/robolectric/Robolectric.java

```diff
@@ -1,20 +1,26 @@
 package org.robolectric;
 
+import static android.os.Build.VERSION_CODES.P;
 import static com.google.common.base.Preconditions.checkState;
 import static org.robolectric.shadows.ShadowAssetManager.useLegacy;
 
 import android.annotation.IdRes;
 import android.app.Activity;
+import android.app.ActivityThread;
+import android.app.AppComponentFactory;
 import android.app.Fragment;
 import android.app.IntentService;
+import android.app.LoadedApk;
 import android.app.Service;
 import android.app.backup.BackupAgent;
 import android.content.ContentProvider;
+import android.content.Context;
 import android.content.Intent;
 import android.os.Bundle;
 import android.os.Looper;
 import android.util.AttributeSet;
 import android.view.View;
+import androidx.annotation.RequiresApi;
```

Comment:
Robolectric shadows should not depend on AndroidX libraries.  This could be `android.annotation.RequiresApi`.

Reply:
Thank you, fixed

Reply:
I see `androidx.annotation` here: https://github.com/robolectric/robolectric/blob/977de00af24994a6a34790544e06d232a5f67bc7/shadows/framework/build.gradle#L65. Is that okay? It transitively appears in `robolectric` module as well.